TunnelVisionLabs.ReferenceAssemblyAnnotator/AnnotatorBuildTask.cs

```diff
@@ -55,8 +55,16 @@ public string OutputPath
             set;
         }
 
+        public string[]? NoWarn
```

Comment:
This would normally be `string`, since `NoWarn` is defined as a comma-separated list and MSBuild only breaks up arrays for semicolons. The C# compiler allows both `,` and `;` as delimiters.